src/ui/components/SidePanel.tsx

```diff
@@ -93,19 +97,6 @@ export default function SidePanel() {
 
   return (
     <div className="flex w-full flex-col gap-2">
-      {shouldShowDevToolsNag(nags, viewMode) && (
```

Comment:
Do we need to remove the import for `shouldShowDevToolsNag` from this file after dropping this code?